Extensions/IISWebAppDeploy/Src/Tasks/SqlDacpacDeploy/task.json

```diff
@@ -0,0 +1,193 @@
+    },
+    {
+      "name": "DatabaseName",
+      "type": "string",
+      "label": "Database Name",
+      "required": true,
+      "groupName": "target",
+      "defaultValue": "",
+      "visibleRule": "TargetMethod = server",
+      "helpMarkDown": "Provide the name of the SQL Server database."
+    },
+    {
+      "name": "SqlUsername",
+      "type": "string",
+      "label": "SQL Username",
+      "required": false,
+      "groupName": "target",
+      "defaultValue": "",
+      "visibleRule": "TargetMethod = server",
+      "helpMarkDown": "If the SQL Server login is specified, it will be used to connect to the SQL Server. The default is Integrated Authentication and uses the machine administrator's credentials."
+    },
+    {
+      "name": "SqlPassword",
+      "type": "string",
+      "label": "SQL Password",
+      "required": false,
+      "groupName": "target",
+      "defaultValue": "",
+      "visibleRule": "TargetMethod = server",
+      "helpMarkDown": "If SQL Server login user name is specified, then provide the SQL Server password. The default is Integrated Authentication and uses the machine administrator's credentials."
+    },
+    {
+      "name": "ConnectionString",
+      "type": "multiLine",
+      "label": "Connection String",
+      "required": true,
+      "groupName": "target",
+      "defaultValue": "",
+      "visibleRule": "TargetMethod = connectionString",
+      "helpMarkDown": "Specify the SQL Server connection string like \"Server=localhost;Database=Fabrikam;User ID=sqluser;Password=password;\"."
+    },
+    {
+      "name": "PublishProfile",
+      "type": "filePath",
+      "label": "Publish Profile",
+      "required": false,
+      "groupName": "target",
+      "defaultValue": "",
+      "helpMarkDown": "Publish profile provide fine-grained control over SQL Server database creation or upgrades. Specify the path to the Publish profile XML file on the target machine or on a UNC share that is accessible by the machine administrator's credentials."
+    },
+    {
+      "name": "AdditionalArguments",
+      "type": "multiLine",
+      "label": "Additional Arguments",
+      "required": false,
+      "groupName": "target",
+      "defaultValue": "",
+      "helpMarkDown": "Additional SqlPackage.exe arguments that will be applied when creating or updating the SQL Server database like, /p:IgnoreAnsiNulls=True /p:IgnoreComments=True. These arguments will override the settings in the Publish profile XML file (if provided)."
+    },
+    {
+      "name": "DeployInParallel",
+      "type": "boolean",
+      "label": "Deploy in Parallel",
+      "defaultValue": "true",
+      "required": false,
+      "groupName": "advanced",
+      "helpMarkDown": "Setting it to true will run the database deployment task in-parallel on the target machines."
+    }
+  ],
+  "instanceNameFormat": "Deploy SQL DACPAC: $(DacpacFile)",
+  "execution": {
+    "PowerShell": {
```

Comment:
We should no longer introduce new Tasks that are based on PowerShell handler. This is getting deprecated in a sprint or so. We should move to PS3.